evergreen.yml

```diff
@@ -138,11 +241,18 @@ tasks:
     commands:
       - func: "install rust toolchain"
       - func: "compile release"
+      - func: "upload artifact"
 
   - name: test-rust
+    depends_on:
+      - name: compile
     commands:
+      - func: "download artifact"
+      - func: "add Windows DLL path"
+      - func: "setup driver on Windows"
       - func: "install rust toolchain"
       - func: "run rust tests"
+      - func: "clean-up driver on Windows"
```

Comment:
I did not find how to have a clean-up function/task which would run even if tests failed, but only after the tests ran, not after every task.

Reply:
There are things called `pre` and `post`

Reply:
https://github.com/10gen/sqlproxy/blob/master/.evg.yml#L661

Reply:
I tried using pre and post, but this is not ideal because it runs before every task (clippy, etc...) and will fails for all tasks which are not test-rust unfortunately.
I went back to Evergreen wiki and I will try to use a task group, with only this task inside. It's a bit of a work-around, but if it works then I will be happy.